packages/office-ui-fabric-react/src/components/pickers/Suggestions/Suggestions.tsx

```diff
@@ -95,60 +69,105 @@ export class Suggestions<T> extends BaseComponent<ISuggestionsProps<T>, ISuggest
       resultsFooter,
       isResultsFooterVisible = true,
       suggestionsAvailableAlertText,
-      suggestionsHeaderText
+      suggestionsHeaderText,
+      suggestionsClassName,
+      theme,
+      styles
     } = this.props;
 
+    // TODO
+    // Clean this up by leaving only the first part after removing support for SASS.
+    // Currently we can not remove the SASS styles from Suggestions class because it
+    // might be used by consumers separately from pickers extending from BasePicker
+    // and have not used the new 'styles' prop. Because it's expecting a type parameter,
+    // we can not use the 'styled' function without adding some helpers which can break
+    // downstream consumers who did not use the new helpers.
+    // We check for 'styles' prop which is going to be injected by the 'styled' HOC
+    // in BasePicker when the typed Suggestions class is ready to be rendered. If the check
+    // passes we can use the CSS-in-JS styles. If the check fails (ex: custom picker),
+    // then we just use the old SASS styles instead.
+    this._classNames = styles
+      ? getClassNames(styles, {
+          theme: theme!,
+          className,
+          suggestionsClassName,
+          forceResolveButtonSelected: this.state.selectedActionType === SuggestionActionType.forceResolve,
+          searchForMoreButtonSelected: this.state.selectedActionType === SuggestionActionType.searchMore
+        })
+      : {
+          root: css('ms-Suggestions', className ? className : '', legacyStyles.root),
```

Comment:
I Don't believe you need the ternary, I think css will just ignore undefined, but I'm not 100% on that.

Reply:
Fixed.